SupportFiles/jira_plugin-install.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Script to download and install JIRA Plugins from other sources
+#
+#################################################################
+# shellcheck disable=SC2086
+PROGNAME="$(basename ${0})"
+SCRIPTHOME="${HOME:-/root}"
+
+JIRADCHOME="/var/atlassian"
+JIRAINSTDIR="/opt/atlassian/jira"
+DBCFGDIR="${JIRADCHOME}/application-data/jira"
+PLUGINS1DIR="${JIRAINSTDIR}/atlassian-jira/WEB-INF/lib"
+PLUGINS2DIR="${DBCFGDIR}/plugins"
+INSTLPLUGINS2DIR="${DBCFGDIR}/plugins/installed-plugins"
+
+# Array of URLs to download plugins (Type 1 or 2)
```

Comment:
Safe to assume that, since the template-mods only reference the location of the script  but not a parseable list of plugin-URLs, that this script is being offered as a starting point rather than "here's your ready-to-go plugin script"?

Reply:
Ultimately, we'll want this to _stay_ unpopulated in this project. This project is meant to be generic. We don't want to force plugins on users of _this_ project.

People using this script will use it as a starting point. They'll copy it down, edit it and store their specific version in another location (S3, Artifactory, a git fork-branch, etc.). In sum, "don't worry about making changes based on feed-back"